authentication/templates/authentication/register.html

```diff
@@ -0,0 +1,153 @@
+                        required>
+                </div>
+
+                <div class="text-center text-danger my-3" id="msg-password"></div>
+
+                <div class="input-group my-3">
+                    <span class="input-group-text bg-primary" id="basic-addon1">
+                        <i class="fas fa-lock text-white"></i>
+                    </span>
+                    <input class="form-control" type="password" name="confirm-password" placeholder="Confirm Password"
+                        id="confirm-password" required>
+                </div>
+
+                <div class="text-center text-danger my-3" id="msg-confirm-password"></div>
+                <input type="submit" class="form-control fw-bold btn btn-primary" value="Register"
+                    onclick="register()"></input>
+                <div class="text-center text-success my-3" id="msg-success"></div>
+                <div class="text-center text-danger my-3" id="msg-password-match"></div>
+            </form>
+
+        </div>
+    </div>
+</body>
+
+<script>
+
+    async function login(data) {
+        //debugger
+        let response = await fetch("{% url 'authentication:create-token' %}", {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(data)
+        });
+        data = await response.json();
+
+        if (response.ok) {
+            sessionStorage.setItem("access", data.access)
+            sessionStorage.setItem("refresh", data.refresh)
+            sessionStorage.setItem("user_id", data.user_id)
+            window.location.href = "{% url 'core:home-view' %}";
+
+        } else {
+            document.getElementById('msg').innerHTML = data.detail;
+        }
+    }
+
+    function clear_all_msg() {
+        document.getElementById('msg-username').innerHTML = "";
+        document.getElementById('msg-password').innerHTML = "";
+        document.getElementById('msg-confirm-password').innerHTML = "";
+        document.getElementById('msg-success').innerHTML = "";
+        document.getElementById('msg-password-match').innerHTML = "";
+    }
+
+    async function register() {
+
+        clear_all_msg();
+        //debugger
+        let username = document.getElementById('username').value;
+        let password = document.getElementById('password').value;
+        let confirmPassword = document.getElementById('confirm-password').value;
+        if (password != confirmPassword) {
+            document.getElementById('msg-password-match').innerHTML = "Password did't match";
+            return;
+        }
+        let formData = new FormData();
+        formData.append('username', username);
+        formData.append('password', password);
+        formData.append('confirm_password', confirmPassword);
+
+        let response = await fetch("{% url 'authentication:register-user' %}", {
```

Comment:
**Take or leave**
In django templates, I'd recommend to make a common/shared template for URLs, something like `urls.html`, 
and inside it we can define all the URLs in script tag, this way, our all URLs will be kept at one place and will be easy to maintain.